src/components/ha-markdown-element.ts

```diff
@@ -1,15 +1,22 @@
-import { customElement, property, UpdatingElement } from "lit-element";
+import { customElement, property, ReactiveElement } from "lit-element";
 import { fireEvent } from "../common/dom/fire_event";
 import { renderMarkdown } from "../resources/render-markdown";
 
 @customElement("ha-markdown-element")
-class HaMarkdownElement extends UpdatingElement {
+class HaMarkdownElement extends ReactiveElement {
   @property() public content?;
 
   @property({ type: Boolean }) public allowSvg = false;
 
   @property({ type: Boolean }) public breaks = false;
 
+  public connectedCallback() {
+    super.connectedCallback();
+    if (!this.shadowRoot) {
+      this.attachShadow({ mode: "open" });
```

Comment:
This didn't have a shadow root on purpose. `<ha-markdown>` is the one with shadow dom. This element was just to render the markdown. 

This causes `<ha-markdown>` styling to not be applied

![image](https://user-images.githubusercontent.com/1444314/118337503-cebec600-b4c8-11eb-90fc-2593ad352891.png)

Reply:
Ok, will fix

Reply:
https://github.com/home-assistant/frontend/pull/9187